### DISCUSSION TITLE
Add relay in custom php [RESOLVED]

### DISCUSSION THREAD

**Discussion by maxencelobry** [cachewerk/relay#76] [Support]
I have no knowledge of this kind of thing, but is it possible to add relay.so to a custom PHP ? 

Let me explain, I'm currently trying to add relay to my php: https://github.com/pmmp/PHP-Binaries (I'm modifying the compile.sh)

And I really can't do it (it keeps telling me that the relay extension is missing because I'm using predis with relay parameters) 

In fact, in compile.sh there's no case where it's approximately the same~ as installing relay, it's mostly makefiles except that relay has only one .so file

**Comment by tillkruss** [comment]
Relay must be loaded via its `relay.so` file and a INI directive.

**Reply by maxencelobry** [reply]
Ik, but only doing that, I get an error: relay.so: undefined symbol: executor_globals

**Reply by maxencelobry** [reply]
Hi ! I have found the problem, thanks for help
